components/style_derive/to_computed_value.rs

```diff
@@ -25,7 +25,9 @@ pub fn derive(input: DeriveInput) -> Tokens {
             }
             quote! { ::std::clone::Clone::clone(#binding) }
         } else {
-            where_clause.add_trait_bound(&binding.field.ty);
+            if !attrs.ignore_bound {
```

Comment:
This is only to avoid recursive bounds, right? Ir so, couldn't we just check it? Also, not super-confident of the name, perhaps something else should be chosen instead... @nox?

Reply:
That sounds more complex and we only have one or two cases of recursive types so I think this is good as is.